TripPlanner - Project/TripPlannerFrontend/src/components/Layout/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { DefaultLayout } from './DefaultLayout';
+import { Typography } from '../Typography/Typography';
+import Logo from '../Branding/Logo';
+
+const testUser = {
+  isLogged: false,
+  name: 'Bill',
+  surname: 'Gates',
+};
+
+export default function Header() {
+  const [users, setUsers] = useState(testUser);
```

Comment:
`users` (plural) implies that there's multiple users. Use just `user, setUser`

Reply:
ok, zmienione